packages/react/time-series/src/TimeSeries.tsx

```diff
@@ -203,6 +196,32 @@ export function TimeSeries(props: TimeSeriesProps) {
         }
       }
 
+      if (chartRef.current) {
+        const datasetRef = chartRef.current.data.datasets[0]
+
+        chartRef.current.data.labels = labels
+        datasetRef.data = values
+        if (chartRef.current.options.layout?.padding) {
+          chartRef.current.options.layout.padding = styles?.canvas?.padding
+        }
+        chartRef.current.options.scales = isFormatted || isStatic ? customFormatScales : autoFormatScales
+
+        datasetRef.type = variant
+        datasetRef.backgroundColor =
+          styles?.bar?.backgroundColor || styles?.line?.backgroundColor || defaultStyles.bar.backgroundColor
```

Comment:
Let's not use styles of other elements like `bar`/`line`. Let's do just <prop> || <defaultValue>:

```suggestion
          styles?.bar?.backgroundColor || defaultStyles.bar.backgroundColor
```